packages/react/src/components/layout/GridLayout.tsx

```diff
@@ -43,7 +50,7 @@ export function GridLayout({ filter, filterDependencies, ...props }: GridLayoutP
       gridEl.current.style.setProperty('--lk-col-count', cols.toString());
       gridEl.current.style.width = `${widthAdjust}%`;
     }
-  }, [width, height, participants, gridEl.current]);
+  }, [width, height, participants, gridEl]);
```

Comment:
did you double check that this doesn't have re-render side effects?
I think that was the initial reason why I ignored the eslint warning here for `.current`

Reply:
I didn't -> reverted 👍